### ISSUE TITLE
Unresponsive netopeer-cli

### ISSUE THREAD

**Issue by Sameer-Chouksey** [CESNET/netopeer#187]
Hello all,

I am trying to connect netopeer client to a netopeer server. The server is running in the NXP, LS1028ARDB device. I am running the client in an Ubuntu machine. 

When I try to connect the client with the server see following response on the server side:

_netopeer-server[2044]: Adding new event (5)
netopeer-server[2044]: Created session 1 for user 'root' (UID 0) (recovery)
netopeer-server[2044]: New server session for 'root' with ID 1_

Below is the response that I get on the client side. 

_netopeer-cli
netconf> verbose
Verbose level set to VERBOSE
netconf> connect --port 830 --login root 169.254.67.231
libnetconf VERBOSE: Publickey athentication 
libnetconf VERBOSE: No key pair specified.
libnetconf VERBOSE: Password authentication (host 169.254.67.231, user root)
root@169.254.67.231 password:_

After this the terminal doesn't give the control back (no netconf> ). not able to issue any command after this.  

Please check and help me know what I am missing. I am also adding complete logs from the server side in a file, please check.

Thanks,
Sameer 
[Netopeer_server_logs.txt](https://github.com/CESNET/netopeer/files/4102604/Netopeer_server_logs.txt)

**Comment by michalvasko** [comment]
Hi,
just from the log it may be possible you are using a wrong *libssh* version with a regression bug. Try the release 0.9.2 instead.

Regards,
Michal

**Comment by Sameer-Chouksey** [comment]
Hello Michal,

Thanks for the prompt response. 
I have tried with libssh version 0.9.2, 0.9.3 as well as 0.8.0. Same behavior in both the cases. 

Regards,
Sameer

**Comment by michalvasko** [comment]
Right, in that case I am unable to help you. We are not supporting this old *netopeer* for quite a few years, please try [netopeer2](https://github.com/cesnet/netopeer2).

Regards,
Michal
